flake.nix

```diff
@@ -4,7 +4,7 @@
   inputs = {
     nixpkgs.url =
       "github:NixOS/nixpkgs?rev=b8c367a7bd05e3a514c2b057c09223c74804a21b";
-    nixpkgs-terraform.url = "github:manveru/nixpkgs/updated-terraform-providers";
+    nixpkgs-terraform.url = "github:johnalotoski/nixpkgs-terraform/iohk-terraform-2021-06";
```

Comment:
hostile takeover :)

Reply:
I mean we could probably move it into the org instead, but it's not really a blocker.

Reply:
A truce between manveru and I has been declared! ;)
Moved to IOHK nixpkgs.